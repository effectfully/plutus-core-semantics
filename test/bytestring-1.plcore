(program

(module Foo (imported) (exported () ()) (local () ())
(define b1
  (builtin take
   5 (builtin drop
     1 (builtin concatenate
       #123456 #789ABC))))

(define b2
  (let
    (lam x (builtin concatenate x #1)) f
      [f #ABC]))
)

)
