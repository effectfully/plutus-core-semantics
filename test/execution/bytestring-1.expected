<T>
  <k> . </k>
  <lenv> .Map </lenv>
  <genv> ( Foo . b1 ) |-> #23456 ( Foo . b2 ) |-> #ABC1 </genv>
  <modules>
    <Module>
      <curr> 0 </curr>
      <name> Foo </name>
      <dataCons> .TyExpList </dataCons>
      <decl> .NameList </decl>
      <importedModules> .ModNmList </importedModules>
      <exportTy> .TyExpList </exportTy>
      <exportTm> .NameList </exportTm>
    </Module>
  </modules>
</T>
